src/Tests/Microsoft.NET.PerformanceTests/BuildPerf.cs

```diff
@@ -75,7 +75,26 @@ public void BuildWebLarge(ProjectPerfOperation operation)
 
             TestProject(Path.Combine(testDir.Path, "mvc"), "Build Web Large", operation);
         }
+   
+        [CoreMSBuildOnlyTheory]
+        [InlineData(ProjectPerfOperation.CleanBuild)]
+        [InlineData(ProjectPerfOperation.BuildWithNoChanges)]
+        public void BuildWebLarge30(ProjectPerfOperation operation)
+        {
+            string sourceProject = Path.Combine(TestContext.GetRepoRoot(), ".perftestsource/PerformanceTestProjects/WebLarge");
+            var testDir = _testAssetsManager.CreateTestDirectory("WebLarge30", identifier: operation.ToString());
+            Console.WriteLine($"Mirroring {sourceProject} to {testDir}...");
+            FolderSnapshot.MirrorFiles(sourceProject, testDir.Path);
+            Console.WriteLine("Done");
+            if (File.Exists(Path.Combine(testDir.Path, "mvc", "mvc.csproj")))
+            {
+                File.Delete(Path.Combine(testDir.Path, "mvc", "mvc.csproj"));
+            }
+            File.Move(Path.Combine(testDir.Path, "mvc", "mvc30-csproj.txt"), Path.Combine(testDir.Path, "mvc", "mvc.csproj"));
```

Comment:
Do we really need to do this copy around stuff? I have some project manipulation code in this repo. Is it only for projects that are being created?

Reply:
@livarcocc 
This netcoreapp2.0 project already exists @ https://github.com/dotnet/BuildPerformanceTestAssets/tree/master/PerformanceTestProjects/WebLarge

It would be more straight-forward to make a '3.0' copy at the BuildPerformanceTestAssets repo - but that would require a double local copy when we pull it down to run tests.

An alternative method would be to string replace "netcoreapp2.0" -> "netcoreapp3.0"